packages-oss.yml

```diff
@@ -103,7 +103,6 @@ build-command: VERSION_PKG_PATH=github.com/hashicorp/vault/vendor/github.com/has
 # is a useful data source for building CI/CD pipelines.
 packages:
   - inputs: { GOOS: darwin, GOARCH: amd64 }
-  - inputs: { GOOS: darwin, GOARCH: arm64 }
```

Comment:
Accidental deletion?

Reply:
I suppose this is from a bad merge as well? Fixed in 861abee11 and followed by yet another `make packages`